packages/terra-consumer-nav/src/Nav.jsx

```diff
@@ -62,6 +63,44 @@ const propTypes = {
     ),
   ),
   /**
+   * An array of items to be displayed as help menu content.
+   */
+  helpItems: PropTypes.arrayOf(PropTypes.shape(
```

Comment:
Based on a previous code review comment added the prop shapes both on component and sub components.